utils/mox-pkcs11/Makefile

```diff
@@ -0,0 +1,44 @@
+#
+## Copyright (C) 2020, 2025 CZ.NIC z.s.p.o. (https://www.nic.cz/)
+#
+## This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+# #
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=mox-pkcs11
+PKG_VERSION:=2.0
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://gitlab.nic.cz/turris/mox-pkcs11.git
+PKG_MIRROR_HASH:=424b5247288310c6a71b7babed201dc153c58fca73241530254d2cbc84f1b4ef
+PKG_SOURCE_VERSION:=v$(PKG_VERSION)
+
+PKG_MAINTAINER:=CZ.NIC <packaging@turris.cz>
+PKG_LICENSE:=GPL-3.0-or-later
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/mox-pkcs11
+  TITLE:=MOX PKCS11
```

Comment:
I think it would be appropriate to add here the category and section where we can find the package in ``make menuconfig``. And if you could also add the URL, that would be great, of course.